STREAMING/SetPedPopulationBudget.md

```diff
@@ -8,6 +8,10 @@ ns: STREAMING
 void SET_PED_POPULATION_BUDGET(int p0);
 ```
 
+```
```

Comment:
Manual descriptions shouldn't be written in \`\`\`. This is only used for NativeDB imports.